src/main/java/fly/wild/wizards/serverside/ServerConnnector.java

```diff
@@ -82,100 +48,61 @@ public ServerConnnector (TLSConfiguration tlsConfiguration, OneWayTLSConfigurati
+		// Generate a key pair
+		DefaultOperationRequestBuilder generateCertificateBuilder = buildGenerateCertificateBuilder ();
+		// Persist the key pair		
+		DefaultOperationRequestBuilder storeCertificateBuilder = buildStoreCertificateBuilder ();
+		//Create a key manager
+		DefaultOperationRequestBuilder keyManagerBuilder = buikdKeyManagerBuilder ();
+		//Configure SSL Context
+		DefaultOperationRequestBuilder serverSSLContextBuilder = buildServerSSLContextBuilder ();
 		
-		/*
-		 * /subsystem=elytron/key-store=exampleKeyStore:store()
-		 */
-		ModelNode storeCertificate = new ModelNode();
-		storeCertificate.get(ClientConstants.OP).set(STRING_OPERATION_STORE);
-		storeCertificate.get(ClientConstants.OP_ADDR).add(STRING_SUBSYSTEM, STRING_SUBSYSTEM_ELYTRON);
-		storeCertificate.get(ClientConstants.OP_ADDR).add(STRING_KEY_STORE_ATTRIBUTE, this.genKeyStoreName);
-		
-		
-		
-		
-		/*
-		 * /subsystem=elytron/key-manager=exampleKeyManager:add(
-		 * key-store=exampleKeyStore,credential-reference={clear-text=secret})
-		 */
-		ModelNode keyManager = new ModelNode ();
-		keyManager.get(ClientConstants.OP).set(ClientConstants.ADD);
-		keyManager.get(ClientConstants.OP_ADDR).add(STRING_SUBSYSTEM, STRING_SUBSYSTEM_ELYTRON);
-		keyManager.get(ClientConstants.OP_ADDR).add(STRING_KEY_MANAGER_ATTRIBUTE, this.genKeyManagerName);
-		keyManager.get(STRING_KEY_STORE_ATTRIBUTE).set(this.genKeyStoreName);
-		keyManager.get(STRING_CREDENTIAL_REFERENCE_ATTRIBUTE).get(STRING_CLEAR_TEXT_ATTRIBUTE).set(STRING_CLEAR_TEXT_VALUE_KEY_STORE);
-		
-		/*
-		 * /subsystem=elytron/server-ssl-context=examplehttpsSSC:add(key-manager=exampleKeyManager, protocols=["TLSv1.2"])
-		 */
-		ModelNode serverSSLContext = new ModelNode ();
-		serverSSLContext.get(ClientConstants.OP).set(ClientConstants.ADD);
-		serverSSLContext.get(ClientConstants.OP_ADDR).add(STRING_SUBSYSTEM, STRING_SUBSYSTEM_ELYTRON);
-		serverSSLContext.get(ClientConstants.OP_ADDR).add(STRING_SERVER_SSL_CONTEXT_ATTRIBUTE,this.genServerSSLContext);    		
-		serverSSLContext.get(STRING_KEY_MANAGER_ATTRIBUTE).set(this.genKeyManagerName);		
-				
 		try {
-			this.client = ModelControllerClient.Factory.create (InetAddress.getByName(this.ipAddress), 
-					DMR_PORT);
 			
-			client.execute(new OperationBuilder(keyStore).build());
-			client.execute(new OperationBuilder(generateCertificate).build());
-			client.execute(new OperationBuilder(storeCertificate).build());
-			client.execute(new OperationBuilder(keyManager).build());
-			client.execute(new OperationBuilder(serverSSLContext).build());
+			ModelControllerClient client = ModelControllerClientFactory.CUSTOM.getClient(connectionAddress, null, false, null, false, 0, null, null, null);
 			
-		} catch (IOException e) {
+			final ModelNode keytore = client.execute(keyStoreBuilder.buildRequest());
+			this.logMessage.append("Creating a key store\n");
+			this.logMessage.append(keytore.toString()+"\n");
```

Comment:
just a total minor, this can be `this.logMessage.append(keytore.toString()).append("\n");` instead. Same below. It is better to use append calls everywhere instead of having `+` concatenation inside the append method

Reply:
done globally